lib/logstash/codecs/identity_map_codec.rb

```diff
@@ -100,7 +111,15 @@ def initialize(codec)
     @identity_map = ThreadSafe::Hash.new &method(:codec_builder)
     @max_identities = MAX_IDENTITIES
     @evict_timeout = EVICT_TIMEOUT
-    @cleaner = MapCleaner.new(self, CLEANER_INTERVAL)
+
+    @cleaner = PeriodicRunner.new(self, CLEANER_INTERVAL, :map_cleanup)
```

Comment:
Can we use the `cleaner_interval` method as the common way to define the cleaner?